### PR TITLE
Request model

### PR DESCRIPTION
**Pull request by romkatsu**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | request model

Example integration https://github.com/romkatsu/yii-api-demo/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,111 @@
...
-The package ...
+Request model is used to simplify working with request data. It allows you to decorate data for easy retrieval and automatically validate it when needed.
 
 ## Installation
 
 The package could be installed with composer:
 
 ```
-composer install yiisoft/_____
+composer install yiisoft/request-model
```
Comment (roxblnfk) [inline README.md:24]:
```suggestion
composer require yiisoft/request-model
```

Comment (romkatsu) [reply README.md]: Fixed

**Review by xepozz — APPROVED** [review]
Looks good to me.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Great job! Thank you.

**Pull request merged**